puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/pr_checkout.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -xe
+
+cd plugin/
```

Comment:
This shouldn't be in a generic "pr_checkout", perhaps "plugin_pr_checkout" or use the root directory.